MinimedKit/PumpEvents/PumpAlarmPumpEvent.swift

```diff
@@ -8,11 +8,22 @@
 
 import Foundation
 
+public enum PumpAlarmType: Int {
+    case BatteryOutLimitExceeded = 3
+    case NoDelivery              = 4
+    case BatteryDepleted         = 5
+    case DeviceReset             = 16
+    case ReprogramError          = 61
+    case EmptyReservoir          = 62
+    case UnknownType             = -1
```

Comment:
Something along the lines of this:
https://github.com/ps2/rileylink_ios/blob/dev/MinimedKit/Messages/MySentryPumpStatusMessageBody.swift#L38